bot/cogs/eval.py

```diff
@@ -174,16 +174,15 @@ def _format(self, inp, out):  # (str, Any) -> (str, discord.Embed)
 
     @group(name='internal', aliases=('int',))
     @with_role(Roles.owner, Roles.admin)
-    async def internal_group(self, ctx):
+    async def internal_group(self, ctx: discord.Context) -> None:
```

Comment:
It's `discord.ext.commands.Context`; I'd recommend importing as `Context`
```suggestion
    async def internal_group(self, ctx: Context) -> None:
```